### PR TITLE
Update to version 3.28.0

### PR DESCRIPTION
**Pull request by jjhelmus**

### CODE REVIEW

**Comment by jjhelmus** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: Why the move to pypi?

Comment (jjhelmus) [reply recipe/meta.yaml]: PyPI publishes the sha256 for files and has some guarantees about the immutability of the files.  GitHub archives can and do change and I know of no manner of finding the hash besides downloading the file.

Comment (CJ-Wright) [reply recipe/meta.yaml]: Could we use this? I think the releases downloads are stable
https://github.com/fonttools/fonttools/releases/download/3.28.0/fonttools-3.28.0.zip

Comment (CJ-Wright) [reply recipe/meta.yaml]: Thank you!

**Comment by CJ-Wright** [comment]
This looks great! Thank you very much @jjhelmus

**Pull request merged**